vscode/src/debugger/session.ts

```diff
@@ -62,7 +62,7 @@ export class QscDebugSession extends LoggingDebugSession {
   ) {
     super();
 
-    this.program = vscode.Uri.parse(this.config.program).path;
+    this.program = fileAccessor.resolvePathToUri(this.config.program);
```

Comment:
I had to explicitly change this from a Uri to a string to fix github.dev. Are you sure this won't break opening a repo on GitHub again?

Reply:
It all works as far as I can test on mac, local web, vscode.dev side loading, wsl, and windows.